authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerHiveIT.java

```diff
@@ -138,6 +138,9 @@ public void testOnRoleCreated() {
     RoleEntity role = mock3TableRole(currentFunName());
     Assertions.assertTrue(rangerAuthHivePlugin.onRoleCreated(role));
     verifyRoleInRanger(rangerAuthHivePlugin, role);
+
+    // Repeat to create the same to verify the idempotent operation
+    Assertions.assertTrue(rangerAuthHivePlugin.onRoleCreated(role));
```

Comment:
Why is it judging true or false here?
Please add, separately, the test cases you added that throw the 4 exceptions.

Reply:
I test it can be called by multiple times.